main.go

```diff
@@ -175,3 +174,13 @@ func retrieveTables(schemadir string) (map[string]*ast.StructType, map[*ast.Stru
 
 	return tables, funcMap, ti, nil
 }
+
+func trimAnnotation(comment string) string {
+	if trimmed := strings.TrimPrefix(comment, "//+table:"); trimmed != comment {
+		return trimmed
+	}
+	if trimmed := strings.TrimPrefix(comment, "// +table:"); trimmed != comment {
+		return trimmed
+	}
+	return comment
```

Comment:
one of prefix cases comparing to a comment is better; like table driven test.

```suggestion
	prefixies := []string{"//+table:", "// +table:"}
	for _, prefix := range prefixies {
		if trimmed := strings.TrimPrefix(comment, prefix); trimmed != comment {
			return trimmed
		}
	}
	return comment
```

Reply:
:+1: